lametro/models.py

```diff
@@ -132,29 +132,34 @@ def latest_council_seat(self):
     @property
     def committee_sponsorships(self):
         '''
-        should we omit 'board of directors' as a committee?
-        they are primary sponsor of a lot of activity, so the first
-        10 items in this feed looks p much the same for every member
-        and is thus not that useful, whereas committee-specific
-        feeds are a bit more distinctive (altho committees seem
-        to sponsor much less legislation... with the most recent
-        being from february - do they only meet at certain times?)
-        '''
-
-        m = self.memberships.all().filter( _organization__classification='committee').distinct('_organization')
+        This property returns a list of ten bills, which have recent actions 
+        from the organizations that the person has memberships in.
 
-        if m:
-            oids = [o._organization_id for o in m]
-
-            leg = []
-            for id_ in oids:
-                try:
-                    committee = Organization.objects.filter(ocd_id=id_)[0]
-                    leg += committee.recent_activity
-                except IndexError: # handle errant oid in my db; pls resolve
-                    pass
-            return leg
-        return None
+        Organizations do not include the Board of Directors.
+        '''
+        query = '''
+            SELECT bill_id 
+            FROM councilmatic_core_bill as bill
+            JOIN councilmatic_core_action as action
+            ON bill.ocd_id=action.bill_id
```

Comment:
this is gonna sound extremely nitpicky from the person who wrote the slapdash code on the left, but, if you don't mind, can you put spaces around the equal signs? it's a lil easier to read.

Reply:
ahaha. happily!